specificatie/persoon.yaml

```diff
@@ -191,6 +213,10 @@ components:
           type: boolean
         leeftijd:
           type: boolean
+        gemeenteVanInschrijving:
```

Comment:
gemeenteVanInschrijving kan niet in onderzoek staan. die mag eruit

Reply:
@JohanBoer je gaf net aan dat je alle opmerkingen verwerkt hebt, maar heb je denk ik over het hoofd gezien